docs/DeveloperGuide.md

```diff
@@ -557,3 +557,66 @@ testers are expected to do more *exploratory* testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Effort**
+
+The creation of GreenMileageEfforts came with a great deal of effort from each and every one of the team members. Since the start of the project we have
+made the effort to consistently conduct teams meeting every week, most of these meetings are held in a physical setting such that we could communicate our ideas effectively.
```

Comment:
perhaps it should be `team meetings` instead of `teams meeting`?

Reply:
also since the meeting was in the past, should it be

most of these meetings _were_ held in a physical setting